docs/articles/sdk/configure-qtcreator.rst

```diff
@@ -0,0 +1,52 @@
+Configuring QtCreator to use the SDK
+====================================
+
+After the installation of the SDK and sourcing environment, there are couple of
+steps to do for running QtCreator in target environment. This roadmap assumes
+that SDK and QtCreator should be installed and environments are sourced already.
+
+Adding a new kit
+--------------------
```

Comment:
Can we have screenshots to support this section? It will make it easier for users to understand what to do.

Reply:
I can create and find a way to add them. good idea.